minkindr/include/kindr/minimal/implementation/quat-sim-transform-inl.h

```diff
@@ -0,0 +1,42 @@
+#ifndef KINDR_MINIMAL_IMPLEMENTATION_QUAT_SIM_TRANSFORM_INL_H_
+#define KINDR_MINIMAL_IMPLEMENTATION_QUAT_SIM_TRANSFORM_INL_H_
+
+namespace kindr {
+namespace minimal {
+
+template <typename Scalar>
+QuatSimTransformTemplate<Scalar>::QuatSimTransformTemplate() : scale_A_B_(1.) {}
+
+template <typename Scalar>
+QuatSimTransformTemplate<Scalar>::QuatSimTransformTemplate(
+    const Transform& T_A_B, const Scalar scale_A_B_)
+    : T_A_B_(T_A_B), scale_A_B_(scale_A_B_) {}
```

Comment:
So e.g. eclipse interestingly makes the right interpretation - maybe an initializer list thing. But I'll definitely change the name :) 
![screenshot from 2016-05-10 09 36 54](https://cloud.githubusercontent.com/assets/4147796/15138867/ee8d0782-1692-11e6-915b-7eec9618c831.png)